style.css

```diff
@@ -0,0 +1,444 @@
+}
+
+.header__menu-item{
+    display: inline-block;
+    list-style:none;
+    text-decoration:none;
+    margin-left:10px;
+    padding-left:4px;
+}
+
+
+.header__menu-item a{
+    text-decoration: none; 
+    color:#FBFEF9;
+    font-size: 12px;
+    margin-left:9.5px;
+}
+
+.navigate{
+    display:flex;
+}
+
+.fa{
+    padding: 15px;
+    font-size: 30px;
+    width: 50px;
+    text-align: center;
+    text-decoration: none;
+    margin: 5px;
+    display:flex;
+}
+
+
+.fa:hover {
+    opacity: 0.7;
+}
+
+.fa-facebook {
+    /* background: #3B5998; */
+    color: white;
+}
+
+.fa-linkedin{
+    /* background-color:#0078BB; */
+    color: white;
+}
+
+.fa-github{
+    /* background-color: white; */
+    color:white;
+
+}
+
+.social_media_links{
+    display:flex;
+    list-style:none;
+    border: 1px solid;
+    justify-content: center;
+    background-color:#1C3144;
+
+}
+
+
+footer{
+    /* display:flex; */
+    flex-shrink: 0;
+    margin-top:64px;
+    position:fixed;
+    bottom:0;
+    left:0;
+    width:100%;
+}
+
+/* ********** porfoilio css ************ */
+
+.projects h2{
+    margin-bottom:10px;
+}
+
+.projects__header__1{
+    text-decoration: underline;
+}
+.projects__header__2{
+    margin-top:25px;
+    text-decoration: underline;
+}
+.project-1-link{
+    display:inline-block;
+    padding: 5px;
+    padding-bottom: 10px;
+}
+
+.project-2-link{
+    display:inline-block;
+    padding: 5px;
+    padding-bottom:10px;
+}
+
+.project-1-link a{
+    text-decoration:none;
+    color:#FBFEF9;
+}
+
+.project-2-link a{
+    text-decoration:none;
+    color:#FBFEF9;
+}
+
+.project-1-live{
+    border-right: 1px solid white;
+    padding-right: 20px;
+}
+
+.project-2-live{
+    border-right: 1px solid white;
+    padding-right: 20px;
+}
+
+.about-project-1{
+    font-weight:bold;
+    margin-top:10px;
+    margin-bottom:5px;
+}
+
+.about-project-2{
+    font-weight:bold;
+    margin-top:10px;
+    margin-bottom:5px;
+}
+
+/* ************ resume css ********** */
+.main_resume{
+    padding-left:5px;
+}
+ul{
+    list-style-position: inside;
+}
+.page_title{
+    margin-top: 5px;
+    margin-bottom: 10px;
+}
+
+.resume_intro_phone{
+    margin-bottom: 10px;
+}
+
+.resume_intro{
+    margin-bottom: 25px;
+}
+
+.emploment_history_header{
+    margin-bottom: 15px;
+    border: 3px groove black;
+}
+
+.employment_history_cfa{
+    font-size: 1.2rem;
+}
+
+.employment_history_cfa_jobTitle{
+    margin-bottom: 20px;
+}
+
+.employment_history_jobDescription{
+    font-weight: bold;
+    text-decoration: underline;
+}
+
+.education_history_header{
+    margin-top: 15px;
+    margin-bottom: 15px;
+    border: 3px groove black;
+}
+
+.education_history_riverside{
+    margin-bottom: 10px;
+    text-decoration: underline;
+}
+
+.education_history_momentum{
+    margin-top:15px;
+    margin-bottom: 10px;
+    text-decoration: underline;
+}
+
+/* *************** page break Home page *********** */
+
+@media only screen and (min-width: 426px){
```

Comment:
Between 426px and about 600px, your projects page looks pretty wonky. I would take a second look at it to see if you could improve how the layout looks then.